schema/4.0.0/schema.md

```diff
@@ -1112,6 +1112,71 @@ See also `default_embedding` in `uns`.
 </tbody></table>
 <br>
 
+
+### {category}_colors
+
+<table><tbody>
+  <tr>
+    <th>Key</th>
+      <td>
+        {category}_colors where {category} MUST be the name of a <code>category</code> data type column in <code>obs</code> that<br> is annotated by the data submitter or curator. The following columns that are annotated by CELLxGENE<br> Discover MUST NOT be specified as {category}:<br><br>
```

Comment:
The multi-use of `category` here made me have to think really hard.
I think using `column` when referring to the obs field name would alleviate this, and be consistent with the rest of the schema (e.g. "Curators MUST annotate the following _columns_...")

Reply:
Addressed in b1a41adbfffc1a5ee1280deeb13fec4460abad8a.